org.compiere.db.oracle.provider/src/org/adempiere/db/oracle/config/ConfigOracle.java

```diff
@@ -320,22 +320,22 @@ public String test(IDBConfigMonitor monitor, ConfigurationData data)
 		}
 		if (monitor != null)
 			monitor.update(new DBConfigStatus(DBConfigStatus.DATABASE_SERVER, "ErrorDatabaseServer",
-				pass, true, error));
+				pass, false, error));
```

Comment:
you can't just change this to false since if this is an critical error if we are using the traditional approach to connect to Oracle DB.